README.md

```diff
@@ -104,9 +104,15 @@ Keys are used by Liftbridge's log compaction. When enabled, Liftbridge streams w
 
 ```python
 # Publish a message with a key
-client.publish(Message(subject='foo', value='Hello', key='key'))
+client.publish(Message(stream='foo-stream', value='Hello', key='key'))
 ```
 
+Also, it is possible to publish a message to the NATS subject.
+
```

Comment:
A few line here to say about the difference of `stream` and `subject` would be nice